nominatim/api/search/query.py

```diff
@@ -29,6 +29,7 @@ class BreakType(enum.Enum):
     """ Break created as a result of tokenization.
         This may happen in languages without spaces between words.
     """
+    SOFT_PHRASE = ':'
```

Comment:
Can you please add documentation for this new type, just like it is done in the lines above.

Reply:
Thank you so much for your help and comments.
I added the documentation.